src/gluonts/torch/distributions/distribution_output.py

```diff
@@ -106,13 +107,17 @@ class DistributionOutput(Output):
     """
 
     distr_cls: type
+    dim: int = 1
 
     @validated()
     def __init__(self) -> None:
         pass
 
     def _base_distribution(self, distr_args):
-        return self.distr_cls(*distr_args)
+        if self.dim == 1:
+            return self.distr_cls(*distr_args)
+        else:
+            return Independent(self.distr_cls(*distr_args), 1)
```

Comment:
Also here `dim` is unclear: as far as I understand, it controls how many batch dimensions we want to turn into event dimensions. So maybe the following is better?

```python
dim: int = 0
[...]
distr = self.distr_cls(*distr_args)
if self.dim > 0:
    return Independent(distr, self.dim)
return distr
```

Reply:
sorry for the confusion... yes the dim here is the target_dim of the distribution... which is different to the 1 in the Independent which we always want to be 1 i believe